code/lib/codemod/src/transforms/upgrade-deprecated-types.ts

```diff
@@ -33,7 +34,7 @@ export default function transform(info: FileInfo, api: API, options: { parser?:
 
   upgradeDeprecatedTypes(file);
 
-  let output = recast.print(file.path.node).code;
+  let output = formatCsf(csf, { preserveStyle: true }).code;
```

Comment:
Why are we changing the return type of `formatCsf`? I think it's a public API

Reply:
Made a new method `printCsf` indeed, to make no breaking change.